fairlearn/reductions/_moments/utility_parity.py

```diff
@@ -262,8 +275,7 @@ class DemographicParity(UtilityParity):
     (corresponding to the Lagrange multipliers for positive and negative constraints)
     as there are unique values for the sensitive feature.
     The :meth:`signed_weights` method will compute the costs according
-    to Example 3 of
-    `Agarwal et al. (2018) <https://arxiv.org/abs/1803.02453>`_.
+    to Example 3 of `Agarwal et al. (2018) <https://arxiv.org/abs/1803.02453>`_ [1]_. #noqa: E501
```

Comment:
I think you need to add a whitespace after `#` and perhaps an additional one before as well:
`<offending code>  # noqa: E501`
If you got it to pass without that just ignore :-)